sufia-models/app/models/datastreams/fits_datastream.rb

```diff
@@ -144,42 +144,7 @@ def self.xml_template
     http://hul.harvard.edu/ois/xml/xsd/fits/fits_output.xsd",
                version: "0.6.0",
                timestamp: "1/25/12 11:04 AM") {
-        xml.identification {
```

Comment:
This is not a template. It is a directive for parsing xml files.  If the attributes are returning empty strings, it's because the XML tags are empty.  People may be depending on these directive to say, get the md5checksum, now that you've remove them, they'll get a NoMethodError. So if you want to remove them, you should make a replacement method that does exactly what these do and log an error about the method being deprecated and give an alternative method for getting at each thing.